src/TTrigger.cpp

```diff
@@ -26,11 +26,12 @@
 
 
 #include "Host.h"
-#include <QRegularExpression>
 #include "TConsole.h"
 #include "TDebug.h"
 #include "TMatchState.h"
+#include "TMedia.h"
 #include "mudlet.h"
+#include <QRegularExpression>
```

Comment:
Qt class `#include`s need @ahmedcharles 's MSVC debug wrappers:
```suggestion
#include "pre_guard.h"
#include <QRegularExpression>
#include "post_guard.h"
```